orderportal/html/home.html

```diff
@@ -229,8 +216,7 @@
   <div class="modal-dialog modal-lg">
     <div class="modal-content">
       <div class="modal-header">
-        <button type="button" class="close glyphicon glyphicon-remove"
-		data-dismiss="modal"></button>
+		<button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
```

Comment:
Not sure why you weren't using the default close markup from bootstrap here? Feel free to ignore if you had reasons.